send_review_notification/github_services.py

```
@@ -0,0 +1,266 @@
+
+            page_number += 1
+
+
+@check_token
+def create_discussion_comment(org_name: str, repo: str, body: str) -> None:
+    """Creates github discussion on the team_slug with the given title and
+    body.
+    """
+
+    query_category_id = """
+        query($org_name: String!, $repository: String!) {
+            repository(owner: $org_name, name: $repository) {
+                discussionCategories(first: 10) {
+                    nodes {
+                        id
+                        name
+                    }
+                }
+            }
+        }
+    """
+
+    variables = {
+        'org_name': org_name,
+        'repository': repo
+    }
+
+    response = requests.post(
+        GITHUB_GRAPHQL_URL,
+        json={'query': query_category_id, 'variables': variables},
+        headers=_get_request_headers()
+    )
+
+    data = response.json()
+
+    category_id = ''
+    for category in data['data']['repository']['discussionCategories']['nodes']:
+        if category['name'] == 'Reviewer notifications':
+            category_id = category['id']
+            break
+
+    query_discussion_id = """
+        query ($org_name: String!, $repository: String!, $category_id: ID!) {
+            repository(owner: $org_name, name: $repository) {
+                discussions(categoryId: $category_id, first: 10) {
+                    edges{
+                        node {
+                            id
+                            title
+                        }
+                    }
+                }
+            }
+        }
+    """
+
+    variables = {
+        'org_name': org_name,
+        'repository': repo,
+        'category_id': category_id
+    }
+
+    response = requests.post(
+        GITHUB_GRAPHQL_URL,
+        json={'query': query_discussion_id, 'variables': variables},
+        headers=_get_request_headers()
+    )
+
+    data = response.json()
+
+    # Assuming the particular category will have only one discussion.
```

Please validate this assumption. If it is violated, an error should be thrown (i.e. it should fail noisily) so it's easy to see what went wrong

---

done